tillerinobot/src/main/java/tillerino/tillerinobot/lang/Polish.java

```diff
@@ -0,0 +1,211 @@
+package tillerino.tillerinobot.lang;
+
+import java.util.List;
+import java.util.Random;
+
+import org.tillerino.osuApiModel.Mods;
+import org.tillerino.osuApiModel.OsuApiUser;
+
+import tillerino.tillerinobot.CommandHandler.Action;
+import tillerino.tillerinobot.CommandHandler.Message;
+import tillerino.tillerinobot.CommandHandler.Response;
+
+/**
+ * Polish language implementation.
+ * Pawwit https://osu.ppy.sh/u/2070907 & LilSilv https://github.com/LilSilv https://osu.ppy.sh/users/8488688
+ */
+public class Default extends AbstractMutableLanguage {
```

Comment:
oh and missed this, change `Default` to `Polish` here, it has to match the filename.

Also in `LanguageIdentifier.java` line 12 change `Polski.class` to `Polish.class`

Renaming the file gives headaches :eyes:

Reply:
Done ^^